Documentation/FrameworkOverview.md

```diff
@@ -120,6 +147,7 @@ to. Once connected, the connection's signal is said to be _hot_, and the
 underlying subscription will remain active until _all_ subscriptions to the
 connection are [disposed](#disposables).
 
+<!-- TODO: Remove? Or is there a replacement? -->
```

Comment:
Just for my understanding: The replacement is actually `SequenceType`, but its now directly a part of swift, correct?